README.md

```diff
@@ -123,16 +116,14 @@ on how to write meaningful descriptions for your contributions.
 
 #### Doing upmerges
 
-1) Develop only in the `2.x` branch, unless this feature is only compatible with Neos 4.x, then in `master`
-
-2) To do the upmerge do the following commands
+To do the upmerge run the following commands
```

Comment:
I am fine with this PR but maybe we should explain more why we need to handle it this way. Or should we put more explanation on docs.neos.io for instance?

Reply:
I'm just down merging the readme from master here, it's not a part of what I changed, can be adjusted later.